pkg/apis/operators/shared/shared.go

```diff
@@ -35,3 +36,17 @@ func RemoveFinalizer(objectMeta *metav1.ObjectMeta, deletingFinalizer string) {
 
 	return
 }
+
+// IsObjectInOtherNamespace returns true if the namespace is not the watched
+// namespace of the operator
+func IsObjectInOtherNamespace(namespace string) bool {
+	watchNamespace, err := k8sutil.GetWatchNamespace()
+	if err != nil {
+		return false
```

Comment:
Should we highlight in the function documentation that a failure to get get the watch namespace results in a false result?

Reply:
Sure I can do that.

Reply:
I am now wondering I should make this function return `bool, error` and return an error in the case where we cannot get the watch namespace as that indicates something going wrong fundamentally. At the call site, we would return `nil, error`. I will let others weigh in here too.

Reply:
@shawn-hurley @awgreene I have addressed this. PTAL.